app/models/ability.rb

```diff
@@ -4,7 +4,7 @@ class Ability
   include Hydra::Ability
   include Hyrax::Ability
   include GroupAwareRoleChecker
-  # OVERRIDE: Added custom user ability roles
+  # Add custom ability roles
```

Comment:
I don't see override here, is that on purpose?

Reply:
Yes. Since it's adding behavior, not changing behavior that already exists, Rob and I agreed that it wasn't actually an override (you'll see a few other comments throughout this PR where other "OVERRIDE"s are removed for the same reason)